src/utils/setup/get-related-pages-with-images.js

```diff
@@ -7,7 +7,8 @@ export const getRelatedPagesWithImages = (
     const related = dlv(pageNodes, 'query_fields.related', []);
     const relatedPageInfo = related.map(r => {
         // Handle `reference` and `ref_role` types
-        const target = r.target || r.refuri || r.fileid;
+        const getTargetFromFileid = dlv(r, 'fileid.0', null);
+        const target = r.target || r.refuri || getTargetFromFileid;
```

Comment:
I think we actually never want to use the `target` field here. You can see [here](https://github.com/mongodb/devhub/pull/497/files#diff-444081bac694421430b235147dcaad65L9) that we removed that as a dependency for the DOM. It's mainly as an internal reference field for the parser at this point.

```suggestion
        const target = r.refuri || getTargetFromFileid;
```